ObjectMapper/Core/Mapper.swift

```diff
@@ -334,18 +334,18 @@ extension Mapper {
 	public func toJSONString(_ object: N, prettyPrint: Bool = false) -> String? {
 		let JSONDict = toJSON(object)
 		
-        return Mapper.toJSONString(JSONDict, prettyPrint: prettyPrint)
+        return Mapper.toJSONString(JSONDict as Any, prettyPrint: prettyPrint)
```

Comment:
That's not quite accurate. Some of the JSON objects are `[String: Any]` and others are `[[String: Any]]`